src/components/clusterWizard/constants.ts

```diff
@@ -0,0 +1,13 @@
+// export const CLUSTER_STEP_DETAIL = 'cluster-detail';
+export const CLUSTER_STEP_CONFIGURATION = 'cluster-configuration';
```

Comment:
Do we need these contstants? Would defining a type and enforce it where needed be enough?
```
type ClusterSteps = 'cluster-configuration' | 'networking' | ...
```